hyprbars/Makefile

```diff
@@ -1,5 +1,5 @@
 CXXFLAGS = -shared -fPIC --no-gnu-unique -g -std=c++2b -Wno-c++11-narrowing
-INCLUDES = `pkg-config --cflags pixman-1 libdrm hyprland pangocairo libinput libudev wayland-server hyprland`
```

Comment:
Is removing `hyprland` intended?

Reply:
correct me if i'm wrong, but it said hyprland twice so i assumed that that was a mistake

Reply:
@fufexan

Reply:
Ah, missed that last night. My bad.